src/components/BigRadio/BigRadio.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react'
+// @ts-ignore
+import BigRadioStyles from './BigRadio.module.css'
+import { BigRadioContext } from './BigRadioContext'
+
+interface InputProps {
+    label: string
+    value: string
+    description?: string
+    disabled?: boolean
+    id?: string
+    name?: string
+    checked?: boolean
+    width?: string
+    image?: string
+    onChange?(x: React.ChangeEvent<HTMLInputElement>): void
+    onFocus?(x: React.FocusEvent<HTMLInputElement>): void
+    size?: 'tiny' | 'small' | 'medium' | 'large' | 'xlarge'
+  }
+  interface GroupProps {
+    allowedValues?: any
+    wrap?: boolean
+    checkboxes?: any
+    id?: any
+    layout?: 'horizontal' | 'vertical'
+    error?: any
+    descriptionText?: any
+    label?: any
+    labelOptional?: any
+    name?: any
+    type?: any
+    transform?: any
+    value?: any
+    className?: any
+    children?: React.ReactNode
+    options?: Array<InputProps>
+    onChange?(x: React.ChangeEvent<HTMLInputElement>): void
+    size?: 'tiny' | 'small' | 'medium' | 'large' | 'xlarge'
+  }
+
+  const BigRadioGroup = ({
+    id,
+    children,
+    className,
+    type,
+    options,
+    value,
+    name,
+    wrap,
+    onChange,
+    size = 'medium',
+  }: GroupProps) => {
+
+    const [activeId, setActiveId] = useState('')
+    useEffect(() => {
+        setActiveId(value)
+      }, [value])
+    
+    const parentCallback = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setActiveId(e.target.id)
+    }
+
+    const wrapClasses = wrap ? "big-radio-options-wrap" : "big-radio-options"
+
+    return (
+      <BigRadioContext.Provider
+      value={{ parentCallback, type, name, activeId, parentSize: size }}
+    >
+      <fieldset id={id} className={className}>
+
+          <div className={BigRadioStyles['big-radio-container']}>
```

Comment:
.big-radio-container class could be added to `<fieldset>`

also `<fieldset>` styling needs to be reset as it currently uses browser default styling. (has a border and has padding)